tsconfig.base.json

```
@@ -0,0 +1,17 @@
+{
+  "description": "The base configuration for all packages.",
+  "extends": "@vercel/style-guide/typescript",
+  "compilerOptions": {
+    "lib": ["ES2021"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "noImplicitReturns": true,
+    "noUncheckedIndexedAccess": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "sourceMap": false,
+    "strictNullChecks": true,
```

```suggestion
    "strict": true,
```

Lets enable all strict features like we had before

---

It's already in the style guide base config: https://github.com/vercel/style-guide/blob/canary/typescript.json#L9. :)

---

Removed `strictNullChecks`.